src/main/scala/mrpowers/jodie/OperationMetricHelper.scala

```diff
@@ -0,0 +1,179 @@
+package mrpowers.jodie
+
+import mrpowers.jodie.delta.{DeleteMetric, MergeMetric, OperationMetrics, WriteMetric}
+import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
+import org.apache.spark.sql.delta.util.FileNames
+import org.apache.spark.sql.delta.{DeltaHistory, DeltaLog}
+import org.apache.spark.sql.functions.col
+import org.apache.spark.sql.types.{LongType, MapType, StringType, StructType}
+import org.apache.spark.sql.{DataFrame, Encoders, SparkSession}
+
+case class OperationMetricHelper(
+    path: String,
+    startingVersion: Long = 0,
+    endingVersion: Option[Long] = None
+) {
+  val spark                 = SparkSession.active
+  val deltaLog              = DeltaLog.forTable(spark, path)
+  private val metricColumns = Seq("version", "deleted", "inserted", "updated", "source_rows")
+
+  def getMetricsAsDF(): DataFrame = {
```

Comment:
The name getMetricsAsDF seems too general to me, it is hard to understand this function if I only read its name, I would suggest getCountMetrics, let me know your thoughts.

Reply:
done